advanced/dashboard/sso.mdx

```diff
@@ -4,7 +4,8 @@ description: "Customize how your team can login to your admin dashboard"
 ---
 
 <Info>
-  SSO functionality is available on our [Enterprise plan](https://mintlify.com/pricing?ref=sso). Please{" "}
+  SSO functionality is available on our [Enterprise
+  plan](https://mintlify.com/pricing?ref=sso). Please{" "}
```

Comment:
I'm not sure this formatting was intentional, but should render just fine as Markdown.